ui/app/src/components/shared/AssetList.vue

```diff
@@ -4,7 +4,7 @@
       <AssetItem class="token" :symbol="item.asset.symbol" />
       <div class="amount">{{ formatNumber(item.amount) }}</div>
       <div class="action">
-        <slot v-if="!!item.amount" :asset="item"></slot>
+        <slot v-if="Number(item.amount.toFixed(6)) > 0" :asset="item"></slot>
```

Comment:
This may break app if Token has less than 6 decimals. Will you test it by changing a token in `assets.ethereum.ropsten.json` to have 5 decimals?